lib/elixir/lib/list.ex

```diff
@@ -108,11 +108,11 @@ defmodule List do
 
   ## Examples
 
-      iex> List.delete([1, 2, 3], 1)
-      [2, 3]
+      iex> List.delete([4, 5, 6], 4)
+      [5, 6]
 
-      iex> List.delete([1, 2, 2, 3], 2)
-      [1, 2, 3]
+      iex> List.delete([4, 5, 5, 6], 5)
+      [4, 5, 6]
```

Comment:
Wdyt about using something other than numbers in the list? We could have something like:

```elixir
iex> List.delete([:a, :b, :c], :a)
[:b, :c]

iex> List.delete([:a, :b, :b, :c], :b)
[:a, :b, :c]
```